### PR TITLE
INT-236: Wikia-in-your-lang pt/pt-br bug

### PR DESCRIPTION
**Pull request by mari-mcmurtrie**
This bug is related with CE-1220 fix. We should not have ignored "br" part of "pt-br", rather, check "pt" part of "pt-br" then decide to request Wikia-in-your-lang.

Removed unnecessary code from Javascript and php files. Added a new function to check if we need to show WIYL. 

Ticket: https://wikia-inc.atlassian.net/browse/INT-236

### CODE REVIEW

**Comment by mari-mcmurtrie** [comment]
@adamkarminski will you please CR? You are the most familiar with this code (previous related change you made is here: https://github.com/Wikia/app/commit/d575aeafafa7e670fd948cd264b31744e8fe78d4). 
Thank you!!

File: extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js [hunk 1]
```diff
@@ -47,6 +45,18 @@ require(
 			}
 		}
 
+		// Per request we should unify dialects like pt and pt-br
+		// @see CE-1220
+		function shouldShowWikiaInYourLangWithTargetAndContentLanguage(targetLanguage, contentLanguage) {
+			var targetLanguageLangCode = targetLanguage.split('-')[0],
+			    contentLanguageLangCode = contentLanguage.split('-')[0];
+
+			if (targetLanguageLangCode === contentLanguageLangCode) {
```
Comment (adamkarminski) [inline extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js]: You can just use `return targetLanguageLangCode !== contentLanguageLangCode`

**Comment by mari-mcmurtrie** [comment]
@adamkarminski Thanks, that is right! I changed code as you suggested. How does this look?

**Comment by adamkarminski** [comment]
LGTM, merge when tested.

**Comment by mari-mcmurtrie** [comment]
Thanks @adamkarminski !!!
We tested in sandbox. Test case is documented in ticket.

**Pull request merged**